src/core/utils/url.tsx

```diff
@@ -8,7 +8,11 @@ export const useUrls = () => {
   const { data } = useSelector((state: RootState) => state.url);
   const urls = useMemo(() => turnUrlStringsIntoObjects(data), [data]);
 
-  return (name: string) => {
+  return (name: string, returnFalseIfUndefined = false) => {
+    if (returnFalseIfUndefined) {
+      return urls[name] || false;
+    }
+
     if (!urls[name]) {
       throw new Error(`The url ${name} is not defined`);
     }
```

Comment:
This is just a comment and something we should be aware of:
We do [the check](https://github.com/danskernesdigitalebibliotek/dpl-react/blob/e4eaa0d779fe702f97319fe2253b10f5dec658da/src/core/utils/text.tsx#L116) differently in `useText`.
So it would be nice to do it in the same manner. In that way we would expect both systems to behave the same way.